packages/forms/src/FormFieldInput.tsx

```diff
@@ -83,6 +87,7 @@ export const FormFieldInput = <
         value={value}
         name={name}
         id={id}
+        required={hasError}
```

Comment:
Why do we set `required` when we have an error? react-hook-form and Yup control `Input`, I don't know if makes sense to change this state.